dae/dae/genomic_resources/tests/test_cli_hist.py

```diff
@@ -1,209 +1,209 @@
-    assert not (path / GR_CONTENTS_FILE_NAME).exists()
-
-
-def test_resource_histograms_need_update_message(
-        proto_fixture, dask_mocker, capsys, caplog):
-    path, _proto = proto_fixture
-    # Given
-    assert not (path / "one/histograms").exists()
-
-    # When
-    with caplog.at_level(logging.INFO):
-        cli_manage([
-            "resource-histograms", "--dry-run",
-            "-R", str(path), "-r", "one"])
-
-    # Then
-    captured = capsys.readouterr()
-    assert captured.out == ""
-    assert captured.err == ""
-    assert caplog.record_tuples == [
-        ("dae.genomic_resources.histogram",
-         logging.INFO,
-         "resource <one> histograms "
-         "[{'score': 'phastCons100way', 'bins': 100}] need update"),
-    ]
-
-
-def test_repo_histograms_need_update_message(
-        proto_fixture, dask_mocker, tmp_path, capsys, caplog):
-    path, _proto = proto_fixture
-    # Given
-    assert not (path / "one/histograms").exists()
-
-    # When
-    with caplog.at_level(logging.INFO):
-        cli_manage([
-            "repo-histograms", "--dry-run", "-R", str(path)])
-
-    # Then
-    captured = capsys.readouterr()
-    assert captured.out == ""
-    assert captured.err == ""
-    assert caplog.record_tuples == [
-        ("dae.genomic_resources.histogram",
-         logging.INFO,
-         "resource <one> histograms "
-         "[{'score': 'phastCons100way', 'bins': 100}] need update"),
-    ]
-
-
-def test_resource_histograms_no_update_message(
-        proto_fixture, dask_mocker, tmp_path, capsys, caplog):
-    path, _proto = proto_fixture
-    # Given
-    assert not (path / "one/histograms").exists()
-    cli_manage([
-        "resource-histograms", "-R", str(path), "-r", "one"])
-    assert (path / "one/histograms").exists()
-    _, _ = capsys.readouterr()
-
-    # When
-    with caplog.at_level(logging.INFO):
-        cli_manage([
-            "resource-histograms", "--dry-run",
-            "-R", str(path), "-r", "one"])
-
-    # Then
-    out, err = capsys.readouterr()
-    assert out == ""
-    assert err == ""
-    assert caplog.record_tuples == [
-        ("dae.genomic_resources.histogram", logging.INFO,
-         "histograms of <one> are up to date")
-    ]
-
-
-def test_repo_histograms_no_update_message(
-        proto_fixture, dask_mocker, tmp_path, capsys, caplog):
-    path, _proto = proto_fixture
-    # Given
-    assert not (path / "one/histograms").exists()
-    cli_manage([
-        "repo-histograms", "-R", str(path)])
-    assert (path / "one/histograms").exists()
-    _, _ = capsys.readouterr()
-
-    # When
-    with caplog.at_level(logging.INFO):
-        cli_manage([
-            "repo-histograms", "--dry-run", "-R", str(path)])
-
-    # Then
-    out, err = capsys.readouterr()
-    assert out == ""
-    assert err == ""
-    assert caplog.record_tuples == [
-        ("dae.genomic_resources.histogram", logging.INFO,
-         "histograms of <one> are up to date")
-    ]
+# import os
```

Comment:
Delete these tests if they are not relevant anymore.

Reply:
Done